nrfx_config_nrf5340_network.h

```diff
@@ -36,17 +36,20 @@
 #error "Do not include this file directly, only through nrfx_config.h."
 #endif
 
+#define NRF_APPMUTEX   NRF_PERIPH(NRF_APPMUTEX)
+
 /*
- * The MDK provides macros for accessing the peripheral register structures
- * by using their secure and non-secure address mappings (with the names
- * containing the suffix _S or _NS, respectively). Because the nrfx drivers
- * use the macros without any suffixes, you must translate the names.
- * The following section provides configuration for the name translation.
+ * The following section provides the name translation for peripherals with
+ * only one type of access available. For these peripherals, you cannot choose
+ * between secure and non-secure mapping.
  */
-#define NRF_ACL        NRF_ACL_NS
+#if defined(NRF_TRUSTZONE_NONSECURE)
```

Comment:
I don't think this "ifdef is applicable, here,  @anangl 
nRF5340 NETWORK does not have Secure domain.